src/html/partials/footer.html

```diff
@@ -0,0 +1,6 @@
+<footer>
+  <h2>
+    <span>An Exploration by</span>
+    <span><img src="../../logos/Viget_logo.svg" alt=""></span>
```

Comment:
You maaaay not need the span around the image, but it's not hurting anything. Also add "Viget" to the `alt` attribute.